graphics/opencv3/Portfile

```diff
@@ -53,9 +54,19 @@ platform macosx {
     }
 }
 
-worksrcdir          ${subport}-${version}
+#------------------------------------------------------------------------------
+# Subport name used for staging files, etc.
+#
+# This is needed to support Python subports, whose names have no relation to
+# the actual physical file layout.
+#------------------------------------------------------------------------------
+set physical_subport_name \
```

Comment:
Why not just use `${name}` everywhere? The subport names are `${subport}`, so this is extra variable name is unnecessary and not consistent with standard Portfile practice, to the best of my knowledge.

Reply:
Ports opencv3 and opencv4 will soon be combined into a port named 'opencv', with both converted to subports. And once that happens, `physical_subport_name` will disambiguate which one we're dealing with for the Python subports: It'll be set to either opencv3 or opencv4. And we won't be able to use `name`, as that will be `opencv`.

Make sense?

Reply:
Of note, I'm not enamored with the name of variable `physical_subport_name`, but couldn't decide on a better choice at the time.

Now that I've had a chance to sleep on it, perhaps it might be more logical to be called `parent_subport_name`? (As the python subports will essentially be logical child subports of opencv3 and opencv4, even though MacPorts technically doesn't support subports of subports. Lol)